packages/website/package.json

```diff
@@ -15,16 +15,20 @@
     "@next/mdx": "^11.1.2",
     "emotion": "^10.0.17",
     "gray-matter": "^4.0.3",
+    "mdx-prism": "^0.3.3",
     "next": "^12.0.7",
+    "next-mdx-remote": "^3.0.8",
     "react": "16.14.0",
     "react-dom": "16.14.0",
     "react-live": "^2.3.0",
+    "rehype-autolink-headings": "^5.0.1",
+    "rehype-slug": "^4.0.1",
     "remark": "^14.0.2",
-    "remark-html": "^15.0.1"
+    "remark-code-titles": "^0.1.1"
```

Comment:
oh I totally missed this in my first review
how much do we trust in a package in v0.1.1? hahahaha

Reply:
Btw, I used remark-html because it was recommend by next.js team in this blog post
https://nextjs.org/blog/markdown#data-source